src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -2,18 +2,91 @@
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int INITIAL_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private static final int GROW_FACTOR = 2;
+    private Node<K,V>[] table;
+    private int size;
+    private int threshold;
+
+    public MyHashMap() {
+        table = new Node[INITIAL_CAPACITY];
+        threshold = (int) (INITIAL_CAPACITY * LOAD_FACTOR);
+    }
+
     @Override
     public void put(K key, V value) {
-
+        if (size > threshold) {
+            transfer();
+        }
+        int hash = getHash(key);
+        int index = getIndex(hash);
```

Comment:
call getHash() inside getIndex()
```suggestion
        int index = getIndex(key);
```